ebpf/tracer.go

```diff
@@ -25,9 +28,17 @@ var (
 	minRequiredKernelCode = linuxKernelVersionCode(4, 3, 0)
 )
 
+const (
+	closedConnCleanupInterval = 10 * time.Second
+)
+
 type Tracer struct {
-	m      *bpflib.Module
-	config *Config
+	m       *bpflib.Module
+	perfMap *bpflib.PerfMap
+	config  *Config
+	// Lock for the closedConns
+	sync.Mutex
```

Comment:
We should give this an explicit name if it serves a specific purpose and not for general locking/unlocking of the tracer